packages/service/src/glint-service.ts

```diff
@@ -211,7 +243,7 @@ export function severityForDiagnostic(ts: TS, diagnostic: ts.Diagnostic): Diagno
 
 function createSyntheticSourceFile(ts: TS, source: GlintSourceFile): ts.SourceFile {
   return Object.assign(
-    ts.createSourceFile(source.filename, source.contents, ts.ScriptTarget.Latest),
+    ts.createSourceFile(source.filename, source.contents, ts.ScriptTarget.Latest, true),
```

Comment:
This makes SourceFile object similar to those we got from LanguageService with getStart(), getText() etc.